pkgs/development/python-modules/optax/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+<<<<<<< HEAD
+=======
+, numpy
+>>>>>>> 609016d06e4 (python3Packages.dm_haiku: init at 0.0.5)
```

Comment:
looks like a bad conflict

Reply:
Does it ever... boy have I messed up with git royally today... 

Pushed an update, thanks for pointing this out.